Makefile

```diff
@@ -195,7 +195,9 @@ $(VM_IMAGE): $(TARFILE) $(NODE_CACHE) packaging/arch/PKGBUILD bots test/vm.insta
 		--script $(CURDIR)/test/vm.install $(TEST_OS)
 
 # convenience target for the above
-vm: $(VM_IMAGE)
+.PHONY: vm
+vm:
+	XZ_OPT=-0 make "$(VM_IMAGE)"
```

Comment:
This looks fine. Can you please send this to starter-kit as well?

Reply:
Was planning to after we sort this out here :+1: